server/Arcadia.Assistant.Calendar/PendingActions/EmployeeVacationsPendingActionsActor.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace Arcadia.Assistant.Calendar.PendingActions
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Akka.Actor;
+    using Akka.Event;
+
+    using Arcadia.Assistant.Calendar.Abstractions;
+    using Arcadia.Assistant.Calendar.Abstractions.Messages;
+
+    public class EmployeeVacationsPendingActionsActor : UntypedActor, ILogReceive
```

Comment:
but it's not only EmployeeVacations, is it?

Reply:
For now it is. But later we can use it also for all other event types easily, so yes, I'll rename it.